lib/puppet/parser/scope.rb

```diff
@@ -820,8 +849,8 @@ def effective_symtable(use_ephemeral)
   #
   # @api public
   #
-  def []=(varname, value, options = {})
-    setvar(varname, value, options = {})
+  def []=(varname, value, options = EMPTY_HASH)
+    setvar(varname, value, options)
```

Comment:
This is different - the original blocks the options from being passed! I think that should be preserved.
Can be done a better way.